### PR TITLE
Centrifuge App: Altair leftover clean up

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...

- adds more altair color improvements
- lazy loads chart components for better chunking

#851 
#852 (partial - currently overflow in tables is an ellispis)

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/components/Web3Provider.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export const Web3Provider: React.FC = ({ children }) => {
       if (!wallet?.installed) throw new Error('Wallet not available')
       setSelectedWallet(wallet)
 
-      // const injected = await (web3EnablePromise || web3Enable('NFT Studio'))
+      await (web3EnablePromise || web3Enable('Centrifuge App'))
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/Web3Provider.tsx:99]: @onnovisser I'm not sure why this was commented out, but I needed it to sign transactions

Comment (onnovisser) [reply centrifuge-app/src/components/Web3Provider.tsx:99]: It was replaced by `wallet.enable(config.name)`, so it should actually have been removed. What happens when you try doing an transaction without it?

Comment (sophialittlejohn) [reply centrifuge-app/src/components/Web3Provider.tsx:99]: This was the error:
Screenshot 2022-06-01 at 18 59 33

File: fabric/src/components/Tooltip/index.tsx [hunk 2]
```diff
@@ -19,11 +19,23 @@ const StyledTrigger = styled(Text)`
...
   position: relative;
-  text-decoration: underline dotted;
   cursor: pointer;
   text-align: left;
   font-weight: 400;
   font-family: ${({ theme }) => theme.fonts.standard};
+  position: relative;
+  width: fit-content;
+
+  ::after {
```
Comment (onnovisser) [inline fabric/src/components/Tooltip/index.tsx:29]: can use `text-decoration-color` instead. positioning the line absolutely will look a bit weird if the text wraps.

Comment (sophialittlejohn) [reply fabric/src/components/Tooltip/index.tsx:29]: the problem wasn't actually the color but the space in between the text and the underline, that's why I had to do it with the absolute positioning. Maybe we can prevent wrapping instead? Use the ellipsis here as well 🤔

File: centrifuge-app/src/components/Web3Provider.tsx (hunk 1 above)
Comment (sophialittlejohn) [reply centrifuge-app/src/components/Web3Provider.tsx]: Seems to be working after all 🤯  not sure why, but line has been removed again

**Pull request merged**